src/main/java/com/google/api/generator/engine/writer/ImportWriterVisitor.java

```diff
@@ -180,6 +183,20 @@ public void visit(TryCatchStatement tryCatchStatement) {
     tryCatchStatement.catchVariableExpr().accept(this);
     statements(tryCatchStatement.catchBody());
   }
+  /** =============================== COMMENT =============================== */
+
+  @Override
+  public void visit (LineComment lineComment){
```

Comment:
sure, I thought it would be taken care of by `bazel run :google_java_format`, but rerunning the google-java-format tool (by downloading the package)can catch these space problem. Did you just run the bazel command and it worked? Thanks